### PR TITLE
added signin and login page

### PR DESCRIPTION
**Pull request by folaneem**
Login and SignUp page for the ticketing UI has been added

### CODE REVIEW

File: src/components/homepage/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    }
+
+    handleSignUp = () => {
+        this.setState({loadSignUpPage: true})
+    }
+
+    render() {
+        const {loadLoginPage, loadSignUpPage} = this.state;
+        let pageToRender = <div>Testing Something</div>;
```
Comment (folaraz) [inline src/components/homepage/HomePage.js]: Please remove the "Testing Something" and put only an empty div there or null.

File: src/components/homepage/signup/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        title: '',
+        Username: '',
+        firstname: '',
+        lastname: '',
+        DOB: '',
+        gender: '',
+        password: ''
+    }
+    onSubmit = (e) => {
+        console.log(this.state.username);
```
Comment (folaraz) [inline src/components/homepage/signup/SignUp.js]: Please remove all the console.log statements here

File: src/components/homepage/signup/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+import './SignUp.css';
+
+class SignUp extends Component {
+
+    state = {
+        title: '',
+        Username: '',
+        firstname: '',
+        lastname: '',
+        DOB: '',
```
Comment (folaraz) [inline src/components/homepage/signup/SignUp.js]: write DOB in full i.e dateOfBirth. When someone checks this they might not know what DOB means.

File: src/components/homepage/signup/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+
+import './SignUp.css';
+
+class SignUp extends Component {
+
+    state = {
+        title: '',
+        Username: '',
```
Comment (folaraz) [inline src/components/homepage/signup/SignUp.js]: always use camel case when naming a variable

File: src/components/homepage/signup/SignUp.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+        firstname: '',
+        lastname: '',
```
Comment (folaraz) [inline src/components/homepage/signup/SignUp.js]: came case for this as well.

File: src/components/homepage/signup/SignUp.js [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+                                    <select id="gender" value={gender}
+                                            onChange={this.handleGenderChange}>
+                                        <option value="" disabled selected>Select your Gender</option>
+                                        <option value="male">Male</option>
+                                        <option value="female">Female</option>
+                                        <option value="other">Other</option>
+                                    </select>
+
+                                    {/*<input id='occupation' name='gender' type='gender' value={gender}*/}
+                                    {/*       placeholder='Gender'*/}
```
Comment (folaraz) [inline src/components/homepage/signup/SignUp.js]: Do we need this?

File: src/components/homepage/signup/SignUp.js [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+                                    <input id='username' name='username' type='username' value={username}
+                                           placeholder='Username'
+                                           onChange={this.handleUsernameChange}/></td>
+                                <td><input id='firstname' name='firstname' type='firstname' value={firstname}
+                                           placeholder='Firstname'
+                                           onChange={this.handleFirstnameChange}
+                                /></td>
+                            </tr>
+                            <tr>
+                                <td><input id='lastname' name='lastname' type='lastname' value={lastname}
```
Comment (folaraz) [inline src/components/homepage/signup/SignUp.js]: Please put a space between last and name in the place holder
do the same for firstname

File: src/components/homepage/login/Login.js [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+
+class Login extends Component {
+    state = {
+        email: '',
+        password: ''
+    }
+
+    onSubmit = (e) => {
+        console.log(this.state.email);
```
Comment (folaraz) [inline src/components/homepage/login/Login.js]: Please remove the console. log statements

File: src/components/homepage/login/Login.css [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+    outline: none;
+    background: #3e0071;
+}
+
+.LoginForm button:focus {
+    outline: none;
+    background: #3e0071;
+}
+
+
```
Comment (folaraz) [inline src/components/homepage/login/Login.css]: Remove all these whitelines

**Review by folaraz — CHANGES_REQUESTED** [review]

File: src/components/homepage/signup/SignUp.js (hunk 3 above)
Comment (folaneem) [reply src/components/homepage/signup/SignUp.js]: done

File: src/components/homepage/HomePage.js (hunk 1 above)
Comment (folaneem) [reply src/components/homepage/HomePage.js]: done

**Review by folaraz — APPROVED** [review]

**Pull request merged**